diff --git a/node_modules/vue-material-year-calendar/src/App.vue b/node_modules/vue-material-year-calendar/src/App.vue
index c5d453f..9df4725 100644
--- a/node_modules/vue-material-year-calendar/src/App.vue
+++ b/node_modules/vue-material-year-calendar/src/App.vue
@@ -119,28 +119,3 @@ export default {
 }
 </script>
 
-<style lang="stylus">
-*
-  margin 0
-  padding 0
-  box-sizing border-box
-#app
-  padding 60px
-  background-color #eaeaea
-.your_customized_wrapper_class
-  background-color: #0aa
-  color: white
-  &.red
-    background-color: #a00
-    color: white
-    &:after
-      background-image url('./assets/baseline-remove_circle-24px.svg')
-      background-size 100% 100%
-  &.blue
-    background-color: #0000aa
-    color: white
-  &.your_customized_classname
-    background-color: yellow
-    color: black
-
-</style>
diff --git a/node_modules/vue-material-year-calendar/src/components/MonthCalendar.vue b/node_modules/vue-material-year-calendar/src/components/MonthCalendar.vue
index 0bcfb9f..f9c5f6b 100644
--- a/node_modules/vue-material-year-calendar/src/components/MonthCalendar.vue
+++ b/node_modules/vue-material-year-calendar/src/components/MonthCalendar.vue
@@ -1,6 +1,7 @@
 <template>
 <div class="c-wrapper">
-  <div class="calendar"
+  <v-card class="calendar rounded-xl"
+          elevation="7"
     @mouseup="mouseUp"
     @mouseleave.stop="mouseUp"
   >
@@ -8,14 +9,18 @@
     <div class="calendar__body">
       <div v-for="(day, key) in 7" :key="`title${day}`" class="calendar__day day__weektitle" :style="{fontSize: weekTitleFontSizeAdjustLang}">{{ showDayTitle(key) }}</div>
       <div v-for="(dayObj, key) in showDays" class="calendar__day" :key="`day${key}`">
-        <div
-          @mouseover="dragDay(dayObj)"
-          @mousedown="mouseDown(dayObj)"
-          class="day"
-          :class="classList(dayObj)">{{ dayObj.value }}</div>
+        <v-tooltip top>
+          <template v-slot:activator="{ on }">
+            <div
+              class="day"
+              v-on="on"
+              :class="classList(dayObj)">{{ dayObj.value }}</div>
+          </template>
+          <span>{{dayObj.title}}</span>
+        </v-tooltip>
       </div>
     </div>
-  </div>
+  </v-card>
 </div>
 </template>
 
@@ -114,6 +119,7 @@ export default {
         if (typeof date === 'string') {
           oDate = {
             date: date,
+            title: date.title,
             className: this.activeClass
           }
         } else if (typeof date === 'object') {
@@ -127,6 +133,7 @@ export default {
         let activeArrayKey = (activeDate % 7) - 1 + firstDay + 7 * row
         this.showDays[activeArrayKey].active = true // to array index
         this.showDays[activeArrayKey].className = oDate.className
+        this.showDays[activeArrayKey].title = oDate.title
       })
     },
     showDayTitle (day) {
@@ -183,7 +190,6 @@ export default {
     year (val) {
       this.initCalendar()
     },
-    // 外層來的資料有變化時
     activeDates (after, before) {
       this.initCalendar()
     }
@@ -200,10 +206,9 @@ export default {
 .c-wrapper
   padding: 10px
 .calendar
-  background-color #fff
+
   min-height 295px
   text-align center
-  color rgba(#353C46, .8)
   border-radius 2px
   min-width 0
   position relative
@@ -221,7 +226,6 @@ export default {
     display flex
     align-items center
     justify-content center
-    border-bottom 1px solid rgba(#C4C4C4, 0.3)
     font-size 18px
     height: 50px
     margin-bottom 12px
@@ -240,9 +244,7 @@ export default {
     align-items center
     font-size 16px
     height 31px
-    color #5DB3D4
   .day__weektitle
-    color rgba(#353C46, .8)
   .day
     font-size 14px
     cursor pointer
@@ -265,29 +267,9 @@ export default {
       border-radius 50%
       z-index: 1
       background-color transparent
-    &.calendar--active:after
-      background-image url('../assets/baseline-remove_circle-24px.svg')
-      background-size 100% 100%
     &:not(.calendar__day--otherMonth):hover
       background-color rgba(#666, 0.1)
       border-radius 5px
-    &.calendar--active
-      background-color rgba(#FFBABA, .5)
-      color #BCBCBC
-      &.info
-        background-color rgba(#17a2b8, .8)
-        color #fff
-        &:after
-          background-image url('../assets/RecordIt.svg')
-          background-size 100% 100%
-      &.warning
-        background-color rgba(#ffc107, .7)
-        color #fff
-        &:after
-          background-image url('../assets/round-warning-24px.svg')
-          background-color rgba(#eaeaea, .3)
-          background-size 100% 100%
   & .calendar__day--otherMonth
-    color: #eaeaea
     cursor: auto
 </style>
diff --git a/node_modules/vue-material-year-calendar/src/components/YearCalendar.vue b/node_modules/vue-material-year-calendar/src/components/YearCalendar.vue
index 96c5805..5f816b8 100644
--- a/node_modules/vue-material-year-calendar/src/components/YearCalendar.vue
+++ b/node_modules/vue-material-year-calendar/src/components/YearCalendar.vue
@@ -2,14 +2,18 @@
   <div class="vue-calendar__container">
     <div v-if="showYearSelector" class="container__year">
       <!-- <span><button @click="addYear(-1)">back</button></span> -->
-      <span
-        v-for="i in 5"
+      <v-btn
+        x-large
+        text
+        v-for="i in 7"
         :key="i"
-        class="year__chooser"
+        :color="activeYear ===  i + activeYear - 4 ? 'primary' : null"
         @click="changeYear(i)"
       >
-        {{ i + activeYear - 3 }}
-      </span>
+        <v-spacer></v-spacer>
+        {{ i + activeYear - 4 }}
+        <v-spacer></v-spacer>
+      </v-btn>
       <!-- <span><button @click="addYear(1)">next</button></span> -->
     </div>
     <div class="container__months" :class="{'hide-weekend': hideWeekend, 'hide-sunday': hideSunday}">
@@ -128,7 +132,8 @@ export default {
           // 若 activeDate 裡的物件少了 className 屬性，就自動填入空字串。否則會變成undefined
           oDate = {
             date: date.date,
-            className: date.className || ''
+            className: date.className || '',
+            title: date.title || 'No data'
           }
         }
         if (dayjs(oDate.date).year() !== parseInt(this.value)) return // 讓2020年1月的資料，不會放到 2019年的1月資料裡
@@ -149,7 +154,7 @@ export default {
   },
   methods: {
     changeYear (idx) {
-      this.activeYear = idx + this.activeYear - 3
+      this.activeYear = idx + this.activeYear - 4
     },
     toggleDate (dateObj) {
       const activeDate = dayjs()
@@ -206,14 +211,8 @@ export default {
   position: relative
   text-decoration: none
   box-shadow: 0 2px 1px -1px rgba(0,0,0,.2), 0 1px 1px 0 rgba(0,0,0,.14), 0 1px 3px 0 rgba(0,0,0,.12)
-  background-color #F6F6F3
   .container__year
-    user-select none
-    height 65px
-    background-color #fff
-    font-size 24px
-    flex 100%
-    text-align center
+    justify-content center
     display flex
     .year__chooser
       height 100%
